mapentity/static/mapentity/mapentity.filter.js

```diff
@@ -87,6 +68,60 @@ MapEntity.TogglableFilter = L.Class.extend({
         return $(this.popover.data('bs.popover').tip);
     },
 
+    load_filter_form: function (mapsync) {
+        var self = this;
+        // On first click on Filter button, load HTML content for form
+        if (!self.loaded_form) {
+            var filter_url = $('#mainfilter').attr('filter-url');
```

Comment:
To avoid multiple `$('#mainfilter')` selectors, you can create a constante before=> `var $mainfilter = $('#mainfilter')` and call this variable